pontoon/sync/formats/compare_locales.py

```diff
@@ -0,0 +1,118 @@
+"""
+Parser and serializer for file formats supported by compare-locales library:
+https://hg.mozilla.org/l10n/compare-locales/
+"""
+from __future__ import absolute_import
+
+import logging
+
+from collections import OrderedDict
+from compare_locales import (
+    parser,
+    serializer,
+)
+
+from pontoon.sync import SyncError
+from pontoon.sync.exceptions import ParseError
+from pontoon.sync.formats.base import ParsedResource
+from pontoon.sync.vcs.models import VCSTranslation
+
+
+log = logging.getLogger(__name__)
+
+
+class CompareLocalesEntity(VCSTranslation):
+    """
+    Represents an entity in a file handled by compare-locales.
+    """
+    def __init__(self, key, string, comment, order):
+        self.key = key
+        self.source_string = string
+        self.source_string_plural = ''
+        self.strings = {None: self.source_string} if self.source_string is not None else {}
+        self.comments = [c.strip() for c in comment.raw_val.split('\n')] if comment else []
```

Comment:
Once `compare-locales` is updated to a version incorporating fix for [bug 1502009](https://bugzilla.mozilla.org/show_bug.cgi?id=1502009), this line should be changed to:
```python
self.comments = comment.raw_val.split('\n') if comment else []
```